cstrike/addons/amxmodx/scripting/map_manager_online_sorter.sma

```diff
@@ -17,11 +17,15 @@ new g_pCvars[Cvars];
 
 new Array:g_aMapsList;
 
+new g_sCurMap[MAPNAME_LENGTH];
```

Comment:
make it static variable inside function
or use MapName from AMXX 1.10 with define